Tests/MongoSwiftSyncTests/MongoCollection+IndexTests.swift

```diff
@@ -96,6 +96,10 @@ final class MongoCollection_IndexTests: MongoSwiftTestCase {
             options.bucketSize = 10
         }
 
+        if try _client!.serverVersion() >= ServerVersion(major: 4, minor: 4, patch: 0) {
+            options.hidden = false
```

Comment:
given conversation I think we should use `true` to ensure we get the option back and can assert on it, otherwise this PR LGTM and I don't need to review again assuming that makes Evergreen all green. nice job getting right back into things 🥳